listenbrainz/webserver/templates/index/index.html

```diff
@@ -43,61 +43,57 @@ <h2>Discover new music</h2>
-                <th>Description</th>
-                <th>Number</th>
-              </tr>
-              </thead>
-              <tbody>
-              {% if listen_count %}
-              <tr>
-                <td>Number of listens</td>
-                <td>{{ listen_count }}</td>
-              </tr>
-              {% endif %}
-              {% if user_count %}
-              <tr>
-                <td>Number of users</td>
-                <td>{{ user_count }}</td>
-              </tr>
-              {% endif %}
-              </tbody>
-            </table>
-          </div>
-          <div class="panel-footer" style="text-align: center">
-            <small>Updated every 5 minutes</small><br />
-            <small><a href="{{ url_for('index.current_status') }}">more on the current status page</a></small>
-          </div>
+    <h2>About us</h2>
+    <p>
+      ListenBrainz is operated by the <a href="https://metabrainz.org">MetaBrainz Foundation</a>
+      which has a long-standing history of curating, protecting and making music data available to the
+      public. MetaBrainz commits itself to safe-guarding the ListenBrainz data indefinitely.
+    </p>
+    <p>
+      We also make this incredibly useful data available to anyone who cares to play with it. You
+      can use our API or our regular public <a href="https://listenbrainz.readthedocs.io/en/production/dev/listenbrainz-dumps/">data dumps</a>.
+      All non-commercial use of this data is free, but commercial users are asked to <a href="https://metabrainz.org/supporters/account-type">support</a>
+      us in order to help fund the project.
+    </p>
+    <p>
+      For more information about this project and its goals, please take a look at our
+      <a href="{{ url_for('index.about') }}">about page</a>.
+    </p>
+    <div class="col-md-7"
+         style="margin-top: 12px; margin-bottom: 12px"
+         id="stats">
+      <div class="card">
+        <div class="panel-heading center-p"><strong>Data Statistics</strong></div>
+        <div class="panel-body">
+          <table class="table table-collapsed">
+            <thead>
+            <tr>
+              <th>Description</th>
+              <th>Number</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% if listen_count %}
+            <tr>
+              <td>Number of listens</td>
+              <td>{{ listen_count }}</td>
+            </tr>
+            {% endif %}
+            {% if user_count %}
+            <tr>
+              <td>Number of users</td>
+              <td>{{ user_count }}</td>
+            </tr>
+            {% endif %}
+            </tbody>
+          </table>
+        </div>
+        <div class="panel-footer" style="text-align: center">
+          <small>Updated every 5 minutes</small><br />
+          <small><a href="{{ url_for('index.current_status') }}">more on the current status page</a></small>
         </div>
       </div>
-
     </div>
-
     <div class="col-md-5"
```

Comment:
I suppose this will be coming in another PR, but I just wanted to mention since we're looking at the homepage that the stats and news panels look like they're missing css styles:
<img width="508" alt="image" src="https://user-images.githubusercontent.com/6179856/171157131-17a3c825-c3cb-46ec-9ba2-495aa6328524.png">

Reply:
Fixed :)